nature.md

```diff
@@ -98,6 +98,8 @@ We mark up some images so that they can be ignored by Assistive Technologies if
 The Nature subscription pages at <a href="https://www.nature.com/nature/subscribe">https://www.nature.com/nature/subscribe</a> currently contain SVG images without text alternatives, and decorative SVG images that are not marked up to be ignored by Assistive Technologies. We are actively working to repair these problems.
 <br><br>
 Third party scripts may inject non-text content without text alternatives, or with inadequate text alternatives. These include images in advertising scripts and in Disqus commenting widgets. When we become aware of deficiencies, we contact third party suppliers to request repairs.
+<br><br>
+When supplied by authors who publish with us, some complex images may lack adequate alternative text, descriptions, and/or captions. Examples may include images of charts, graphs, or maps. Generally, these images are used to supplement text content. A large-scale and currently-active programme of work to improve alt text content submitted by our authors is under way to address this.
```

Comment:
> programme of work to improve alt text content

1. Technically, does this programme solely cover `alt` attribute text or does it more widely include the previously mentioned "alternative text, descriptions, and/or captions"?

And if only `alt` text, will the audience understand the meaning of "alt text"? I think it is the first time it is used in this document.

2. `alt` is of course an HTML attribute, but not marked up as such throughout this document unlike e.g. `lang`. Should we do so, or expand to "alternate text" when not explicitly discussing alt attributes?

Reply:
1. the latter, the implementation relies on all three features simultaneously. Basically our image descriptions are very complicated and often need to contain HTML markup (e.g. for superscript/subscript, or equations etc), so they can't reasonably go in the alt text. Instead we rely on the visible caption and use the alt text and aria descriptions to associate the figures with the caption material. 

2. Hm, yes, an excellent question, and I agree. I think I'll expand the acronym throughout as we're not always referring to the attribute specifically, even outside of this statement 👍

Reply:
Oh turns out there were only those 2 instances of it so that wasn't too painful 😅